src/tests/ftest/control/dmg_pool_query_test.py

```diff
@@ -85,7 +85,7 @@ def test_pool_query_basic(self):
                     "total": self.params.get("total", path="/run/exp_vals/nvme/*")
                 }
             ],
-            "pool_layout_ver": 3,
+            "pool_layout_ver": 4,
             "query_mask": self.params.get("query_mask", path="/run/exp_vals/*"),
             "upgrade_layout_ver": 3,
```

Comment:
Don't we also need to update the expected `upgrade_layout_ver`:
```suggestion
            "upgrade_layout_ver": 4,
```

Reply:
I think you're right. Thanks, Phil.